src/libs/SidebarUtils.js

```diff
@@ -266,6 +266,10 @@ function getOptionData(reportID) {
     result.isThread = ReportUtils.isThread(report);
     result.isChatRoom = ReportUtils.isChatRoom(report);
     result.isTaskReport = ReportUtils.isTaskReport(report);
+    if (result.isTaskReport) {
+        result.isTaskCompleted = ReportUtils.isTaskCompleted(report);
+        result.isTaskAssignee = ReportUtils.isTaskAssignee(report);
```

Comment:
do you think we should add defaults values for these above?

Reply:
We only grab these if the report is a task, so I think they'll either be true or false.

Reply:
Hi ✋ Coming from https://github.com/Expensify/App/issues/22149

We are using the `managerEmail` to check the task assignee and show the green button if it's `isTaskAssignee.` Still, in the sidebar links selector, we didn't specify the property `managerEmail` to the Onyx selector function, making the sidebar not re-rendering on the `managerEmail` change.